### ISSUE TITLE
Could not find 'qebil' to install

### ISSUE THREAD

**Issue by shanif3** [ucsd-cmi/qebil#10]
(qebil) -bash-5.1$ pip install qebil
ERROR: Could not find a version that satisfies the requirement qebil (from versions: none)
ERROR: No matching distribution found for qebil

**Comment by antgonza** [comment]
Hi @shanif3, at the moment there is no pypi release available but you can install the latests version via github, like this:
`pip install https://github.com/ucsd-cmi/qebil/archive/refs/heads/main.zip`. Best,

**Comment by shanif3** [comment]
thanks!
i run: qebil fetch project --ebi-id ERP020591 --output-dir /home --prefix new --download-fastq

and got- 
with issue: MISSING files for PAIRED_Metagenomic_0/ 
with issue: MISSING files for SINGLE_16S_0

why?

**Comment by antgonza** [comment]
The explanation of why this happens is here:
https://qiita.ucsd.edu/static/doc/html/faq.html#some-of-the-studies-have-qiita-ebi-import-as-the-pi-why-is-this

From there:
MISSING: One or more of the fastq files for your study were unavailable for download from EBI/ENA or the downloaded files were found to contain corrupt data and were excluded from our automatic association and processing.

In general, if is just that the files weren't able to download you can try multiple times, until you get all the files.

**Comment by shanif3** [comment]
Thanks for the response!
What do you mean by trying multiple times? If the first run was not successful does that mean that maybe the next run can be successful?

**Comment by antgonza** [comment]
Right, for whatever reason [network outages, multiple download calls, the moon is not in the right location, too much wind ;-)] sometimes the URL for the submitted files is not available from ENA so trying multiple times works. In my limited experience, early in the week seems to work better than the weekend.

**Comment by shanif3** [comment]
interesting ... ;)
bc rn im just trying to download fastqs from random studies and none of them downloading the fastqs. unfortunately :(
at the end i want to have the otu and taxonomy files of the study samples, so am i in the right place?

**Comment by shanif3** [comment]
I want to add, that I have a program that rn can download data from SRA so if I manage to download the fastq from qiita can I do the same process with qiime2 to get the otu and taxonomy files?

**Comment by antgonza** [comment]
I just tried `qebil fetch project --ebi-id PRJEB18644 --verbose --output-dir PRJEB18644 --download-fastq` and it finds the 6 samples but it only downloads 1 file (lane5_NoIndex.R1.ebi.fastq.gz) and I think is because the submitters used for 5 of them:
```
2023-12-15 05:32:53,331 : INFO : root : EBI strategy is:OTHER for SAMEA27288418
2023-12-15 05:32:53,331 : INFO : root : Layout is: SINGLE
```
In other words, the code can't automatically guess what kind of data is this. I think you can use some combinations of parameters to force things - if you are sure of the data you are downloading.

Anyway, I don't see why not using the other tool while we figure this one out; basically the most important thing is the metadata and that the files follow the run_prefix convention (within the preps) so they are easily linkable within Qiita.

**Comment by shanif3** [comment]
I run the same command, and it doesn't download any fastq, not even 1:(
how can i define the data better so it can find the fastq
thanks again!

**Comment by antgonza** [comment]
In the past, we simply ignore hard to download/fix studies/samples - if they are a must for our meta-analysis, we will contact whoever deposited the data and work directly with them to fix them and upload them to Qiita so others can reuse them in the future. However, sometimes, this is not possible. Additionally, we will evaluate the importance of a study based on their results and the number of samples - the lowest we have gone is ~20 samples per study.
